backend/resolvers/mutations/entry.js

```diff
@@ -143,11 +153,28 @@ export function updateEntry (_, args, req) {
   if (req.auth.type !== ADMIN) {
     throw new UserError('Permission Denied')
   }
+  const setEntry = (entry) => {
+    return entry.update(args.input, {
+    fields: ['forSale', 'invited', 'yearLevel',
+      'academicProgram', 'moreCopies', 'excludeFromJudging']
+  })}
+
+  const setSinglePiece = (entry) => {
+    return SinglePiece.findById(entry.pieceId).then( 
+    singlePiece => {return singlePiece.update(args.input, {
+      fields: ['title', 'comment']
+    })}
+  )}
+
   return Entry.findById(args.id)
-    .then(entry => entry.update(args.input, {
-      fields: ['title', 'comment', 'forSale', 'invited', 'yearLevel',
-        'academicProgram', 'moreCopies', 'excludeFromJudging']
-    }))
+    .then((entry) => {return Promise.all([setEntry(entry), setSinglePiece(entry)]).then(
```

Comment:
needs to separate the data passed in from the frontend into the 2 parts its divided into in the backend